upload/wagtail_hooks.py

```diff
@@ -240,3 +253,22 @@ def register_disclosure_url():
         path('upload/',
         include('upload.urls', namespace='upload')),
     ]
+
+@hooks.register('construct_main_menu')
+def add_packages_waiting_for_qa_menu_item(request, menu_items):
+    qam = QAMenuItem(
+        _('Waiting for quality analysis'), 
+        '/admin/upload/package?status__exact=quality-analysis',
+        icon_name='folder',
+        order=300,
+    )
+    
+    mupload = None
+    for m in menu_items:
+        if m.name == 'upload':
+            mupload = m
+            break
+
+    if mupload is not None:
+        if _('waiting-for-quality-analysis') not in [rmi.name for rmi in m.menu.registered_menu_items]:
+            m.menu.registered_menu_items.append(qam)
```

Comment:
@rafaelpezzuto aqui não é melhor usar o for normalmente com break se encontrar rmi.name == _('waiting-for-quality-analysis')

Reply:
Isso foi removido - retomei a abordagem anterior com Proxy de modo que ficou mais elegante.